tests/unit/lms/models/_mixins/public_id_test.py

```diff
@@ -3,18 +3,19 @@
 from h_matchers import Any
 from pytest import param
 
-from lms.db import BASE
+from lms.db import Base
 from lms.models._mixins.public_id import PublicIdMixin
 from lms.models.public_id import InvalidPublicId
 
 
-class ModelTestHost(PublicIdMixin, BASE):
+class ModelTestHost(PublicIdMixin, Base):
     __tablename__ = "model_test_host"
 
     id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
     public_id_model_code = "model_test"
 
 
+@pytest.mark.xdist_group("ModelTestHost")
```

Comment:
This tells pytest to run all the tests in this class in serial with each other (to send them all to the same parallel test runner).